guides/source/association_basics.md

```diff
@@ -110,13 +110,37 @@ class CreateBooks < ActiveRecord::Migration[5.0]
 
     create_table :books do |t|
       t.belongs_to :author, index: true
-      t.datetime :published_at
+      t.string :name
```

Comment:
Looks like when the graphs were regenerated, the arrows are now pointing from the wrong box 😬